test/downstream/odesolve.jl

```diff
@@ -49,3 +49,14 @@ end
             ArrayPartition(zeros(1), [0.75])),
         (0.0, 1.0)),
     Rodas5()).retcode == ReturnCode.Success
+
+function rhs!(duu::VectorOfArray, uu::VectorOfArray, p, t)
+    du = parent(duu)
+    u = parent(uu)
+    du .= u
+end
+
+u = fill(SVector{2}(ones(2)), 2, 3)
```

Comment:
```suggestion
u = fill(ones(2), 2, 3)
```
Would need to add `StaticArrays` as dependency otherwise.

Reply:
Nvm, StaticArrays is already a dependency

Reply:
ohh... I'll just revert the downstream dep

Reply:
wait no, it's only a core dep, not a StaticArrays dep, so this is right.